report/src/main/webapp/app/components/keyboardShortcuts.directive.js

```diff
@@ -183,14 +189,26 @@ define(['angularAMD', 'userSettingsService'], function (angularAMD) {
               '.url-name:not(.is-hidden)').filter(':first').find(
               testUrlSelector);
           if (_.isEmpty(previousTest)) {
-            $previousElement = suiteContainer.find('.test-name');
-            scrollTo($previousElement);
-            $previousElement.click();
-            currentItem.parents('.url-name').removeClass('is-active');
-            suiteContainer.addClass('is-expanded');
+            if (!_.isEmpty($(suiteContainer).prevAll(
+                  '.aside-report:not(.is-hidden)').first()
+                  .find('.test-name'))) {
+              currentTab = $('.nav-tabs > .nav-item').filter('.active').text().replace(/\s/g, '');
```

Comment:
Here you have `active` and in the lines below `is-active` - can we make it consistent?

Reply:
.active is a class for tabs while .is-active is a class for sidebar menu, I didn't modify any html/css structure. I can ask @Skejven what he thinks about it and then maybe change it.

Reply:
@oliwerin what is the problem with existing those two classes?
If you recommend to align them, please do so in a separate pull request.

Reply:
@Skejven It's not a huge issue but it's a class representing certain state and ideally there should be only one - for better consistency, readability and maintainability.